src/mediarithmics/api/core/plugin/PluginPropertyInterface.ts

```diff
@@ -42,6 +44,7 @@ export type PluginProperty =
 
 export type PropertyType =
   'ASSET'
+  | 'ASSET_FOLDER'
```

Comment:
I think that we added `ASSET_FILE` to be consistent. Can you add it as well?

Unfortunately, I don't know how to deprecate something in TypeScript (I searched once and apparently it's not yet supported) so you can keep ASSET.